nostrpostrlib/src/main/java/nostr/postr/Client.kt

```diff
@@ -24,23 +24,24 @@ object Client: RelayPool.Listener {
     internal var relays = Constants.defaultRelays
 
     fun connect(
+        subscriptionId: String = "main-channel",
         filters: MutableList<JsonFilter> = mutableListOf(JsonFilter()),
         relays: Array<Relay> = Constants.defaultRelays
     ) {
         this.filters = filters
         this.relays = relays
         RelayPool.register(this)
-        RelayPool.connect()
+        RelayPool.connect(subscriptionId)
     }
 
     fun disconnect() {
         RelayPool.unregister(this)
         RelayPool.disconnect()
     }
 
-    fun addFilter(filter: JsonFilter) {
+    fun addFilter(requestId: String = "main", filter: JsonFilter) {
         filters.add(filter)
-        RelayPool.sendFilter()
+        RelayPool.sendFilter(requestId)
```

Comment:
`subscriptionId`s only make sense in the presence of multiple of these. This line here isn't telling the RelayPool which filters are to be sent with this id.